dataLayer/modulemanager.datalayer.json

```diff
@@ -1,2 +1,17 @@
-{
-}
+MMdataLayer = [{
+    'languageIsoCode': 'en',
+    'countryIsoCode ': 'US',
+    'originAirportIataCodes ': 'MIA,TXL,SXF',
```

Comment:
This should be an array. You can check the airTRFX proposed data layer of how arrays are supposed to look like

Reply:
I noticed that in the proposed dataLayer this is the format for the variable:
"destinationAirportIataCodes": [
            "BER", "TXL", "SXF"
        ],
Is there a reason why this is like this? Are we going to separate all of them by array?

The format could also be:
'destinationAirportIataCodes': 'LHR,MIL,LAX'
To include all airports in a single value. 

Let me know what you think.

Reply:
I added a comment in the Chat yesterday, but the array follows JSON best practices.